src/Engine/Graphics/Renderer/BaseRenderer.cpp

```diff
@@ -755,6 +755,12 @@ Sizei BaseRenderer::GetPresentDimensions() {
 
 void BaseRenderer::updateRenderDimensions() {
     outputPresent = window->size();
+
+    // make sure that window size is in even dimensions
```

Comment:
Why is this needed?

Reply:
Screen has to be at dimensions that result in a rounding error for this to be required. Maybe you got lucky when you resized.

Reply:
I added a printf to check that the sizes are odd, it did trigger. But I'm on MacOS, this might play into it.